python-sdk/example_dags/example_google_bigquery_gcs_load_and_save.py

```diff
@@ -22,19 +22,28 @@
 from astro.files import File
 from astro.table import Metadata, Table
 
+ASTRO_GCP_CONN_ID = os.getenv("ASTRO_GCP_CONN_ID", "google_cloud_default")
```

Comment:
Always good to see constants extracted to the code and put on top of the file :)